appveyor.yml

```diff
@@ -7,9 +7,10 @@ environment:
 install:
   - ps: Install-Product node $env:nodejs_version
   - set CI=true
-  - npm -g install npm@latest
+  - set AVA_APPVEYOR=true
+  - npm -g install npm@latest || (timeout 30 && npm -g install npm@latest)
   - set PATH=%APPDATA%\npm;%PATH%
-  - npm install
+  - npm install || (timeout 30 && npm install)
```

Comment:
Haha :p

Sounds like me too. I was bumping dependencies on a gadzillion modules this weekend on a super slow connection. `$ david update`, wait 10 minutes, commit, push, `npm publish`, wait 5 minutes, then next module...